usersession/client/client.go

```diff
@@ -534,3 +536,82 @@ func (client *Client) FinishRefreshNotification(ctx context.Context, closeInfo *
 	_, err = client.doMany(ctx, "POST", "/v1/notifications/finish-refresh", nil, headers, reqBody)
 	return err
 }
+
+// AppInstruction is the json representation of possible arguments
+// for the user session rest api to control apps.
+type AppInstruction struct {
+	Action string   `json:"action"`
+	Snaps  []string `json:"snaps,omitempty"`
```

Comment:
@pedronis should this stay per-snap or be more specific and be per-app? If we don't worry about running hooks I think we can scope it down to an app list.

We could go even more generic and allow passing a list of security tag patterns?

Reply:
listing apps would be more consistent with ServiceInstruction, OTOH nothing stops us to add that granularity when it makes sense. For kill this seems the right granularity to me